cli/tox.ini

```diff
@@ -10,7 +10,7 @@ deps =
   -e..
 passenv = DCOS_* CI_FLAGS CLI_TEST_SSH_KEY_PATH CLI_TEST_MASTER_PROXY
 
-[testenv:py34-syntax]
```

Comment:
We should keep syntax checks in one python version, or else we'll have conflicts :)